TTS/bin/train_vocoder_gan.py

```diff
@@ -78,18 +83,18 @@ def format_data(data):
 
 
 def train(
-    model_G,
-    criterion_G,
-    optimizer_G,
-    model_D,
-    criterion_D,
-    optimizer_D,
-    scheduler_G,
-    scheduler_D,
-    ap,
-    global_step,
-    epoch,
-):
+        model_G,
+        criterion_G,
+        optimizer_G,
+        model_D,
+        criterion_D,
+        optimizer_D,
+        scheduler_G,
+        scheduler_D,
+        ap,
+        global_step,
+        epoch,
+        ):
```

Comment:
This wasn't corrected by `black` as well

Reply:
If it is not probably it fits the formatting convention since black is a deterministic as oppose to yapf that we used previously.

Reply:
Yeah, weirdly it seems to be :thinking: